gpflow/expectations.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 the GPflow authors.
```

Comment:
I kinda have the feeling that this file could benefit from splitting into different files as you have already done by splitting it into "sub-sections".
I'm just raising that comment here so it will trigger opinions from the other guys